content/en/blog/general/introducing-reports-server/index.md

```diff
@@ -37,7 +37,8 @@ Reports server stores policy reports and ephemeral reports outside etcd thus red
 
 ### Without Reports Server
```

Comment:
In the table, can we also show the "Number of Pods" or other resources we are creating to generate the policy reports?

That can help make it clear that we are scaling the number of resources and policy reports.